app/assets/javascripts/discourse/app/controllers/preferences/interface.js

```diff
@@ -178,10 +241,33 @@ export default Controller.extend({
         this.set("model.user_option.text_size", this.textSize);
       }
 
-      this.set(
-        "model.user_option.dark_scheme_id",
-        this.enableDarkMode ? null : -1
-      );
+      const makeColorSchemeDefault = this.makeColorSchemeDefault;
```

Comment:
It's a nitpick and up to you, but I tend to avoid creating a variable in this case, I think the fact that you have to grep for two cases instead of one now largely surpasses the 5 chars